ambari-metrics/ambari-metrics-timelineservice/src/main/java/org/apache/ambari/metrics/core/timeline/upgrade/core/MetricsDataMigrationLauncher.java

```diff
@@ -28,17 +28,15 @@
 import org.apache.commons.logging.LogFactory;
 import org.apache.hadoop.conf.Configuration;
 
-import java.io.BufferedReader;
-import java.io.FileInputStream;
-import java.io.FileWriter;
-import java.io.IOException;
-import java.io.InputStreamReader;
+import java.io.*;
```

Comment:
We don't use wildcards in imports:
https://cwiki.apache.org/confluence/display/AMBARI/Coding+Guidelines+for+Ambari

Reply:
fixed